### PR TITLE
[Feature] - 433215 - Audio Recorder

### PR DESCRIPTION
**Pull request by mpamaro**
This feature aims to add the ability to record and send audio messages, although it has some other miscellaneous fixes.

### CODE REVIEW

File: lib/src/widgets/animations/ds_record_status.widget.dart [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    super.key,
+  });
+
+  @override
+  State<DSRecordStatus> createState() => _DSRecordStatusState();
+}
+
+class _DSRecordStatusState extends State<DSRecordStatus> {
+  final shouldFadeIn = RxBool(true);
+  Timer? timer;
```
Comment (RaulRodrigo06) [inline lib/src/widgets/animations/ds_record_status.widget.dart:18]: a suggestion is to use AnimationController then its not necessary to use any timer or a RxBool

**Review by leonardogbr — CHANGES_REQUESTED** [review]

File: lib/src/widgets/buttons/ds_play_button.widget.dart [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
+    super.key,
+    required this.color,
+    required super.onPressed,
+    super.isLoading,
+  }) : super(
+          icon: SvgPicture.asset(
+            'assets/images/play.svg',
+            package: DSUtils.packageName,
+            color: color,
+            height: 24.0,
```
Comment (leonardogbr) [inline lib/src/widgets/buttons/ds_play_button.widget.dart:20]: If I'm not looking wrong, the correct play button size is 32.0 for audio message bubble.

File: lib/src/widgets/buttons/ds_pause_button.widget.dart [hunk 3]
```diff
@@ -1,36 +1,21 @@
...
+    super.key,
+    required this.color,
+    required super.onPressed,
+    super.isLoading,
+  }) : super(
+          icon: SvgPicture.asset(
+            'assets/images/pause.svg',
+            package: DSUtils.packageName,
+            color: color,
+            height: 24.0,
```
Comment (leonardogbr) [inline lib/src/widgets/buttons/ds_pause_button.widget.dart:18]: If I'm not looking wrong, the correct pause button icon size is `32.0` for audio message bubble.

File: lib/src/widgets/animations/ds_record_status.widget.dart (hunk 1 above)
Comment (mpamaro) [reply lib/src/widgets/animations/ds_record_status.widget.dart]: Nice! I've done.

**Review by RaulRodrigo06 — APPROVED** [review]

**Pull request merged**